### PR TITLE
[MaryTTS] Add log files documentation

### PR DESCRIPTION
**Pull request by FlorianSW**

### CODE REVIEW

File: addons/voice/org.openhab.voice.marytts/README.md [hunk 1]
```diff
@@ -25,3 +25,6 @@ marytts:bits1hsmm bits1-hsmm (de)
 
 The MaryTTS service produces audio streams using WAV containers and PCM (signed) codec with 16bit depth.
 The sample frequency depends on the chosen voice and ranges from 16kHz to 48kHz.
+
+## Log files
```
Comment (martinvw) [inline addons/voice/org.openhab.voice.marytts/README.md]: There is a newline missing after this header

File: addons/voice/org.openhab.voice.marytts/README.md [hunk 2]
```diff
@@ -25,3 +25,6 @@ marytts:bits1hsmm bits1-hsmm (de)
...
+The log files of Mary TTS are not bundled with the openHAB log files in `userdata/logs/openhab.log` but are stored in their own log files at `userdata/log/server.log`.
```
Comment (martinvw) [inline addons/voice/org.openhab.voice.marytts/README.md]: AFAIK the linux apt installation does not put the logs in `userdata/logs/` so maybe we should come up with better more generic way to express this, wdyt?

Comment (FlorianSW) [reply addons/voice/org.openhab.voice.marytts/README.md]: Hmm, would something like:

"... with the openHAB log files in the `openhab.log` file of your log directory but are stored in their own log file at `server.log` of your log directory."

be better here?

Comment (FlorianSW) [inline addons/voice/org.openhab.voice.marytts/README.md]: "The log messages of Mary TTS..." would be better, too, I think (will do that in the next commit, once the above thing is clarified :)).

File: addons/voice/org.openhab.voice.marytts/README.md (hunk 1 above)
Comment (FlorianSW) [reply addons/voice/org.openhab.voice.marytts/README.md]: Will fix that together with the thing in L30, once it is clarified :)

**Comment by martinvw** [comment]
Yes your suggestion sounds good 👍

**Comment by martinvw** [comment]
Thanks!

**Pull request merged**